### PR TITLE
Use currency from quote carts market in campaign

### PR DESCRIPTION
**Pull request by tobiasbexelius**
# Jira Issue: [MX-422]

## What?
- This is a bit of a hack, but we don't have a way to easily support this properly atm.

## Why?
-

## Optional checklist
- [ ] Unit tests written
- [x] Tested locally
- [x] Codescouted

[MX-422]: https://hedvig.atlassian.net/browse/MX-422?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by XSwifter — APPROVED** [review]

File: src/resolvers/quote-cart/quoteCart.ts [hunk 1]
```diff
@@ -234,11 +234,15 @@ export const QuoteCartSubset: ResolverSubset = {
         return null
       }
 
+      const incentive = transformIncentive(res.incentive, session.market)
+
+      console.log(`INCENTIVE ${JSON.stringify(incentive)}`)
```
Comment (cpiehl1) [inline src/resolvers/quote-cart/quoteCart.ts:239]: Is this supposed to be here?

Comment (tobiasbexelius) [reply src/resolvers/quote-cart/quoteCart.ts:239]: NO, thanks 😄

File: src/resolvers/campaigns/campaigns.ts [hunk 2]
```diff
@@ -18,18 +18,36 @@ export const CampaignSubset: ResolverSubset = {
         __typename: 'Campaign',
         ...res,
         ownerName: res.campaignOwner,
-        incentive: transformIncentive(res.incentive),
+        incentive: transformIncentive(res.incentive, null),
```
Comment (maksym-krupenko) [inline src/resolvers/campaigns/campaigns.ts:21]: I think using default param like market = null would be suitable here

**Review by maksym-krupenko — APPROVED** [review]

File: src/resolvers/campaigns/campaigns.ts (hunk 2 above)
Comment (tobiasbexelius) [reply src/resolvers/campaigns/campaigns.ts:21]: Cool, did not know ts had that!

**Pull request merged**